reporter/src/main/kotlin/reporters/evaluatedmodel/DependencyTreeNode.kt

```diff
@@ -42,7 +42,7 @@ data class DependencyTreeNode(
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val scopeExcludes: List<ScopeExclude> = emptyList(),
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
-    val issues: List<EvaluatedOrtIssue> = emptyList(),
+    val issues: List<EvaluatedIssue> = emptyList(),
```

Comment:
Commit message: Technically this is also a breaking change for programmatic use as the class is not internal.